<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank Card Number Checker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>631</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="TitleLabel">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>36</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Bank Card  Checker</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="CardNumLabel">
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Card Number: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CardTypeEdit">
         <property name="font">
          <font>
           <family>Papyrus</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ValidateButton">
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Validate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="DisplayMessage">
       <property name="font">
        <font>
         <family>Papyrus</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
